lib/src/utils/parse/parse.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:convert';
+
+  /// Formats CLI arguments to make them JSON parsable.
+  //  Turns [{'asdf':'asdf'}] -> [{"asdf":"asdf"}]
+parseArgsJson(String str) {
+//  Replace all single quotes ' with double quotes " so json.decode is able to read it. 
+    String quoteReplace = str.replaceAll("\'", ('\"'));
+//  Json decode string
+    Map baseMap = json.decode(quoteReplace);
+// Kill trailing white space, convert inputs to string to allow for staticly typed DART methods
```

Comment:
Wow I didn't know u were a murderer... consider changing to Remove. Also Dart not DART and .